packages/kit/src/core/utils.js

```diff
@@ -80,11 +80,18 @@ export function get_mime_lookup(manifest_data) {
 
 /** @param {import('types').ValidatedConfig} config */
 export function get_aliases(config) {
+	/** @type {Record<string, string>} */
 	const alias = {
 		__GENERATED__: path.posix.join(config.kit.outDir, 'generated'),
 		$app: `${get_runtime_path(config)}/app`,
+
+		// $lib should be moved into a default value for `config.kit.alias`
```

Comment:
```suggestion
	// For now, we handle `$lib` specially here rather than make it a default value for
	// `config.kit.alias` since it has special meaning for packaging, etc.
```